home/tests/test_models.py

```diff
@@ -225,16 +250,41 @@ def test_template_not_submitted_with_no_message(
         """
         home_page = HomePage.objects.first()
         main_menu = PageBuilder.build_cpi(home_page, "main-menu", "Main Menu")
-        PageBuilder.build_cp(
-            parent=main_menu,
-            slug="ha-menu",
-            title="HealthAlert menu",
-            bodies=[],
-            whatsapp_template_name="WA_Title_1",
-        )
+        with self.assertRaises(ValidationError):
+            PageBuilder.build_cp(
+                parent=main_menu,
+                slug="ha-menu",
+                title="HealthAlert menu",
+                bodies=[],
+                whatsapp_template_name="WA_Title_1",
+            )
```

Comment:
Having a look at this test in more depth, I think we should rather keep it's original functionality. The reason it is now failing is because it doesn't set a title, which we're now checking for. So I think we should rather add a title (change the `bodies=[]` to `bodies=[WABody("WA Title", [])]`), and then remove the `with self.assertRaises(ValidationError):`

That would then ensure that the title validation check passes, and the test should then pass without raising the ValidationError

Reply:
This new test_template_submitted_with_no_title fails if I don't raise the validation, which is what I should expect. So I added the  `with self.assertRaises(ValidationError):` part so it passes.